src/inc/crsttypes.h

```diff
@@ -1,6 +1,8 @@
+//
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+//
```

Comment:
That probably means we should fix CrstTypeTool or manually edit its output. Rich or Jan might know more precisely what the header requirements are - for myself I just avoid making any changes to them.